program/c/src/oracle/model/test_price_model.c

```diff
@@ -19,13 +19,38 @@ int test_price_model() {
   prng_t _prng[1];
   prng_t * prng = prng_join( prng_new( _prng, (uint32_t)0, (uint64_t)0 ) );
 
-# define N 96
+# define N 192
 
   int64_t quote0 [N];
   int64_t quote  [N];
   int64_t val    [3];
-  int64_t scratch[N];
 
+  /* Brute force validate small sizes via the 0-1 principle. */
+  for( int cnt=0; cnt<=24; cnt++ ) {
+    for( long mask=0L; mask<(1L<<cnt); mask++ ) {
+      for( int i=0; i<cnt; i++ ) quote[i] = (int64_t) ((mask>>i) & 1L);
+
+      memcpy( quote, quote0, sizeof(int64_t)*(size_t)cnt );
```

Comment:
Is `memcpy` here different from the standard library in argument order? It looks like this overwrites the binary mask you just created with what is likely garbage memory. Shouldn't the order be reversed? If I understand the test right this would still pass even if this is backwards but it means the test isn't actually using the 0-1 principle.

Reply:
Oh yeah, my bad here. Thanks for catching it.